STL_Extension/include/CGAL/Named_function_parameters.h

```diff
@@ -21,9 +21,14 @@
 #include <type_traits>
 #include <utility>
 
-#define CGAL_BGL_NP_TEMPLATE_PARAMETERS T, typename Tag, typename Base
-#define CGAL_BGL_NP_CLASS CGAL::Named_function_parameters<T,Tag,Base>
+#define CGAL_BGL_NP_TEMPLATE_PARAMETERS NP_T=bool, typename NP_Tag=CGAL::internal_np::all_default_t, typename NP_Base=CGAL::internal_np::No_property
```

Comment:
It looks a little off in user code:
```
template <typename MACRO1>
void f(const MACRO2& np = get_default())
```
with the seemingly missing default template parameters. I don't see any fix that would not be uglier, though...

Reply:
I did not see any other solution either.